src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -878,9 +882,77 @@ let ``Shows help text for submit command with default target`` () =
                       --output <FriendlyUri|Id>                           The information to show in the output after the job is submitted.
                       --dry-run                                           Validate the program and options, but do not submit to Azure Quantum.
                       --verbose                                           Show additional information about the submission.
-                      -n <n> (REQUIRED)                                   A number.
-                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
-                      --my-cool-bool (REQUIRED)                           A neat bit.
                       -?, -h, --help                                      Show help and usage information"
     let given = testWithTarget "foo.target" "Help"
     given ["submit"; "--help"] |> yields message
+
+[<Fact(Skip="Multiple Entry Points not yet supported in compiler.")>]
```

Comment:
Are you planning to enable these tests? Is the compiler package working?

Reply:
Yes, the test will be enabled.

Reply:
done